File: Knot.py
class Knot(object):

    def __init__(self, id, cKnots):
        self.id = int(id)
        self.cKnots = list(cKnots)

    def getKnotCount(self) -> "int":
        return len(self.cKnots)

    def __str__(self) -> "string":
        return "Knot [cKnots=" + str(self.cKnots) + "]"

File: src/Knot.java
import java.util.List;


public class Knot {

	private List<Integer> cKnots;
	private int id;
	
	public Knot(int id, List<Integer> cKnots)
	{
		this.cKnots = cKnots;
		this.id = id;
	}
	
	public int getId() {
		return id;
	}

	public int getKnotCount()
	{
		return cKnots.size();
	}

	public List<Integer> getcKnots() {
		return cKnots;
	}

	@Override
	public String toString() {
		return "Knot [cKnots=" + cKnots + "]";
	}
	
	
}
